config/initializers/gds_api.rb

```diff
@@ -1,6 +1,7 @@
 require "gds_api/base"
 require "gds_api/asset_manager"
 require "plek"
+require "attachable"
```

Comment:
I think instead of requiring these explicitly you could wrap the code in a [`to_prepare` block](https://guides.rubyonrails.org/v7.0/autoloading_and_reloading_constants.html#autoloading-when-the-application-boots).